RELEASENOTES.md

```diff
@@ -1,6 +1,12 @@
 # Azure Monitor Metrics for AKS clusters
 
 ## Pending
+
+## Release 11-02-2023
+* Linux image - `mcr.microsoft.com/azuremonitor/containerinsights/ciprod/prometheus-collector/images:6.8.1-main-<tbd>`
```

Comment:
will you be updating the version file later @bragi92 ?

Reply:
aah, thanks @rashmichandrashekar ! For some reason I thought the version had already been upgraded. Added it in this PR.

Reply:
Looks like Vishwa had updated it yesterday for the new release with adding the new metric to the keeplist. I think 6.8.1 was right

Reply:
Thanks Grace, I changed it back to 6.8.1